v2/contribute/decisions/oauth/0005-reserved-scopes.mdx

```diff
@@ -35,6 +35,12 @@ Some scopes we've decided to reserve:
 * `supertokens.roles`: access to role information 
 * `supertokens.permissions`: access to permission information
 
+Based on the above scopes we will:
+- Add information mainly into the id token
+- Make this information available through the access token calling a user info endpoint
+- Not add these to the access token (although we will allow that to be customizable)
+- Do not add any information outside of the user id (and sessionHandle) without an appropriate scope, since adding PII should always be opt-in.
```

Comment:
> Add information mainly into the id token

I was under the impression that we will add it to the access and the id token. For example, what if the dev, on their backend, has routes in which they check for roles. If the access token in this case doesn't have roles, then how will this work? Do other providers add info only to the id token and not access token?

Reply:
This is very confusing. I was under the impression that the user would just add `supertokens.roles` and `supertokens.permissions` as scopes and the access token would have the list of roles / permissions for the user (just like the session token has). 

We should discuss this on a call.

Reply:
Please check how other providers do this and then we can add an ADR for this.

Reply:
Other providers by default don't add roles to the access token. Therefore, we should not do that too.